alerting_alert_rule.go

```diff
@@ -25,6 +25,7 @@ type AlertRule struct {
 	For          string            `json:"for"`
 	ForDuration  time.Duration     `json:"-"`
 	Provenance   string            `json:"provenance"`
+	IsPaused     bool              `json:"is_paused"`
```

Comment:
I think the API this library uses is the Provisioning API. The attribute is named `isPaused` in that API.
```suggestion
	IsPaused     bool              `json:"isPaused"`
```

Reply:
👏

Reply:
You are indeed correct, thank you! https://github.com/grafana/grafana/blob/48a374f50b8f85bf0c8058b6c52d8abc230d85e9/pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go#L138